src/components/board/Board.jsx

```diff
@@ -1,3 +1,31 @@
-const Board = () => {};
+import Image from "next/image";
+const Board = ({ name, role, yearMajor, funFact, picture }) => {
+  return (
+    <div className="flex justify-center w-64 h-96 bg-biscuits-yellow shadow-[20px_-15px_0px_rgb(190,156,106)]">
+      <div className="">
+        <Image
+          src={picture}
+          alt="picture"
+          className="aspect-square object-center object-scale-down my-4 mx-10"
+          width={180}
+          height={100}
+        />
+
+        <div className="text-wrap text-center text-biscuits-green-200 font-shrikhand p-1 -translate-y-1 text-3xl ">
+          {name}
```

Comment:
```suggestion
        <div className="text-wrap text-center text-biscuits-green-200 font-shrikhand p-1 -translate-y-1 text-3xl ">
          {name}
        </div>
```
the names, roles, major... etc should be its own tag like this, rather than nested on top of eachother.